src/services/nft-collection.service.ts

```diff
@@ -19,6 +19,9 @@ class NftCollectionService {
   }: GetListNftCollectionDto = {}) {
     const sortValue: string[] = []
     switch (sort) {
+      case NftCollectionSort.PRIORITY:
+        sortValue.push('priority:desc')
+        break
```

Comment:
e đang thấy show priority ở banner là từ bé đến lớn, chắc confirm lại vs Dino là như nào nha anh
```suggestion
      case NftCollectionSort.PRIORITY:
        sortValue.push('priority:asc')
        break
```

Reply:
priority phải là từ lớn => bé chứ đã priority mà bé => lớn e

Reply:
fixed